app/routes/meetings/components/PizzaButton.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from '@webkom/lego-bricks';
+import { Pizza } from 'lucide-react';
+import Tooltip from 'app/components/Tooltip';
+import { MeetingInvitationStatus } from 'app/store/models/MeetingInvitation';
+import type { EntityId } from '@reduxjs/toolkit';
+import type { MeetingInvitationWithUser } from 'app/reducers/meetingInvitations';
+import type { ReactionsGrouped } from 'app/store/models/Reaction';
+
+type Props = {
+  meeting: {
+    reactionsGrouped?: ReactionsGrouped[];
+    invitations: EntityId[];
+    reactions?: { author: { fullName: string }; emoji: string }[];
+  };
+  meetingInvitations: MeetingInvitationWithUser[];
+};
+
+export const PizzaButton = ({ meeting, meetingInvitations }: Props) => {
+  const pizzaUsers: string[] =
+    meeting.reactions
+      ?.filter((reaction) => reaction.emoji === ':pizza:')
+      .map((reaction) => reaction.author.fullName)
+      .sort() ?? [];
+
+  const attendence: string[] = meetingInvitations
+    .filter((invite) => invite.status === MeetingInvitationStatus.Attending)
+    .map((invite) => invite.user.fullName);
+
+  const notPizzaUsers = attendence
+    .filter((name) => !pizzaUsers.includes(name))
+    .sort();
+
+  const pizzaHasBeenReacted = pizzaUsers && notPizzaUsers;
+  const tooltipContent = pizzaHasBeenReacted ? (
+    <div>
+      <h3>Skal ha pizza</h3>
+      {pizzaUsers.map((user) => (
+        <div key={user}>{user}</div>
+      ))}
+      <h3>Skal ikke ha pizza</h3>
+      {notPizzaUsers.map((user: string) => (
+        <div key={user}>{user}</div>
+      ))}
+    </div>
+  ) : undefined;
+
+  return (
+    <Tooltip content={tooltipContent}>
+      <Button>
+        <Pizza /> Pizza-reaksjoner
+      </Button>
+    </Tooltip>
+  );
```

Comment:
It would be really nice if you could press this to react/unreact with pizza emoji!

Reply:
There is a plan to make the button redirect to @nye(2023)s pizza project on a later date. 
I can make the button temporarily react/unreact with the pizza emoji, but feel like having a button changing its purpose later might make it confusing. (It might be smart in case I do not actually ever finish the pizza project). What do you think?

Reply:
Okay, that makes sense. I think I would still prefer if it did react/unreact in the meantime, but it's up to you if you think it's worth implementing just temporarily. Feel free to merge it now if you want:)